src/borg/archiver.py

```diff
@@ -828,6 +828,25 @@ def _delete_archives(self, args, repository):
             if not archive_names:
                 return self.exit_code
 
+        if args.forced == 2:
+            deleted = False
+            for i, archive_name in enumerate(archive_names, 1):
+                try:
+                    del manifest.archives[archive_name]
+                except KeyError:
+                    self.exit_code = EXIT_WARNING
+                    logger.warning('Archive {} not found ({}/{}).'.format(archive_name, i, len(archive_names)))
+                else:
+                    deleted = True
+                    logger.info('Deleted {} ({}/{}).'.format(archive_name, i, len(archive_names)))
+            if deleted and yes('Are you sure? [y/N] '):
```

Comment:
thinking about removing the yes().

the forced == 1 does not ask either and if users add another --force, they already said that they mean it.